README.md

```diff
@@ -2,7 +2,7 @@
 
 Website created using Jekyll, Bootstrap, some plugins and pure jQuery.
 
-Suggestion are welcome!
+Suggestions are welcome!
```

Comment:
Fixed in https://github.com/sugarlabs/www-sugarlabs/commit/8e794f6cf1657aceff129c682335d7a8043db88a